File: tools-manage/invocationlab-admin/src/main/java/org/javamaster/invocationlab/admin/model/erd/CommonErdVo.java
package org.javamaster.invocationlab.admin.model.erd;

import com.alibaba.fastjson.JSONArray;
import lombok.Data;

@Data
public class CommonErdVo {
    private String projectId;
    private String selectDB;
    private String tableName;
    private String sql;
    private String queryId;
    private String type;

    private Boolean explain;

    private Integer page;
    private Integer pageSize;

    private Boolean isExport;

    private JSONArray rows;
}

File: tools-manage/invocationlab-admin-21/src/main/kotlin/org/javamaster/invocationlab/admin/model/erd/CommonErdVo.kt
package org.javamaster.invocationlab.admin.model.erd

import org.javamaster.invocationlab.admin.annos.AllOpen

import com.alibaba.fastjson.JSONArray


@AllOpen
class CommonErdVo {
    var projectId: String? = null
    var selectDB: String? = null
    var tableName: String? = null
    var sql: String? = null
    var queryId: String? = null
    var type: String? = null

    var explain: Boolean? = null

    var page: Int? = null
    var pageSize: Int? = null

    var isExport: Boolean? = null

    var rows: JSONArray? = null
}
